src/components/v5/common/Navigation/ColonySidebar/partials/RouteSection/consts.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  AddressBook,
+  ClockCountdown,
+  FileText,
+  GearSix,
+  Layout,
+  PresentationChart,
+  UsersThree,
+  UserSwitch,
+} from '@phosphor-icons/react';
+
+import {
+  COLONY_ACTIVITY_ROUTE,
+  COLONY_AGREEMENTS_ROUTE,
+  COLONY_BALANCES_ROUTE,
+  COLONY_EXTENSIONS_ROUTE,
+  COLONY_INCOMING_ROUTE,
+  COLONY_MEMBERS_ROUTE,
+  COLONY_PERMISSIONS_ROUTE,
+  COLONY_STREAMING_ROUTE,
+  COLONY_TEAMS_ROUTE,
+  USER_CRYPTO_TO_FIAT_ROUTE,
+} from '~routes';
+
+import { type RouteSectionItemProps } from './partials/RouteSectionItem/types.ts';
+
+export const sidebarNavigationScheme: RouteSectionItemProps[] = [
+  {
+    id: 'dashboard',
+    icon: Layout,
+    translation: { id: 'navigation.dashboard' },
+    path: '/',
+    routeType: 'colony',
+  },
+  {
+    id: 'activity',
+    icon: ClockCountdown,
+    translation: { id: 'navigation.dashboard.activityFeed' },
+    path: COLONY_ACTIVITY_ROUTE,
+    routeType: 'colony',
+  },
+  {
+    id: 'finances',
+    icon: PresentationChart,
+    translation: { id: 'navigation.finances' },
+    routeType: 'colony',
+    path: COLONY_BALANCES_ROUTE,
+    subItems: [
+      {
+        id: 'balances',
+        translation: { id: 'navigation.finances.balances' },
+        path: COLONY_BALANCES_ROUTE,
+        routeType: 'colony',
+      },
+      {
+        id: 'incoming',
+        translation: { id: 'navigation.finances.incoming' },
+        path: COLONY_INCOMING_ROUTE,
+        routeType: 'colony',
+      },
+      {
+        id: 'streaming',
+        translation: { id: 'navigation.finances.streaming' },
+        path: COLONY_STREAMING_ROUTE,
+        routeType: 'colony',
+      },
```

Comment:
Comment this out for now? As it links to a 404
 
<img width="242" alt="Screenshot 2024-08-02 at 14 57 52" src="https://github.com/user-attachments/assets/517cda1b-dca9-429f-b1a5-1a732d1f7350">

Reply:
I was thinking of leaving this here for now. And then in a separate ticket, hide this behind a feature flag because it was suggested in the specs:

![image](https://github.com/user-attachments/assets/c4ec53b7-1342-411c-a695-318d26bb306a)